examples/01_plotting/plot_surf_atlas.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic loading and plotting of a cortical surface atlas.
```

Comment:
Full stop (```.```) is not necessary in the title/headline. I haven't seen anything of that sort with other examples.